[SN-10] Добавлены сервисы для управления постами и комментариями

jbkkkkk opened the pull request.

jbkkkkk committed:
[SN-10]
Добавлены сервисы для управления постами и комментариями

jbkkkkk committed:
[SN-10]
Добавлен AutoMapper, изменены сервисы для комментариев и постов

c23p1 commented on src/Infrastructure/Infrastructure.csproj:
Можно убрать, так как уже есть ссылка на Application

c23p1 committed:
[SN-10] Добавлено исключение "Сущность не найдена"
Добавлен метод GetPostByIdAsync в сервис PostService
Внесены незначительные изменения
